### PR TITLE
Convert Inventory sub-lists to tables

### PR DESCRIPTION
**Pull request by keithjgrant**
##### SUMMARY
Converts Inventory Access, Hosts, Groups, and Sources lists to tables

Addresses #6189

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Comment by nixocio** [comment]
I triggered an e2e test run.

File: awx/ui_next/src/components/ResourceAccessList/ResourceAccessListItem.jsx [hunk 1]
```diff
@@ -67,70 +54,43 @@ function ResourceAccessListItem({ accessRecord, onRoleDelete, i18n }) {
...
+          accessRecord.username
+        )}
+      </Td>
+      <Td dataLabel={t`First name`}>{accessRecord.first_name}</Td>
+      <Td dataLabel={t`Last name`}>{accessRecord.first_name}</Td>
+      <Td dataLabel={t`Roles`}>
+        <DetailList stacked>
+          {userRoles.length > 0 && (
+            <Detail
+              label={i18n._(t`User Roles`)}
```
Comment (nixocio) [inline awx/ui_next/src/components/ResourceAccessList/ResourceAccessListItem.jsx]:
```suggestion
              label={t`User Roles`}
```

File: awx/ui_next/src/components/ResourceAccessList/ResourceAccessListItem.jsx [hunk 2]
```diff
@@ -67,70 +54,43 @@ function ResourceAccessListItem({ accessRecord, onRoleDelete, i18n }) {
...
+              value={
+                <ChipGroup numChips={5} totalChips={userRoles.length}>
+                  {userRoles.map(renderChip)}
+                </ChipGroup>
+              }
+            />
+          )}
+          {teamRoles.length > 0 && (
+            <Detail
+              label={i18n._(t`Team Roles`)}
```
Comment (nixocio) [inline awx/ui_next/src/components/ResourceAccessList/ResourceAccessListItem.jsx]:
```suggestion
              label={t`Team Roles`}
```

**Review by nixocio — COMMENTED** [review]
Overall this PR looks great.  

I noticed that the toggle button is not responsive.

https://user-images.githubusercontent.com/9053044/116571597-3a097500-a8d9-11eb-9586-9d44ee235690.mov

Also, the e2e tests got a few failures that seem to be related to this change.

File: awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupItem.jsx [hunk 3]
```diff
@@ -1,73 +1,57 @@
...
-                <b>{group.name}</b>
-              </Link>
-            </DataListCell>,
-          ]}
-        />
-        <DataListAction
-          aria-label={i18n._(t`actions`)}
-          aria-labelledby={labelId}
-          id={labelId}
+    <Tr id={`group-row=${group.id}`}>
```
Comment (marshmalien) [inline awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupItem.jsx]:
```suggestion
    <Tr id={`group-row-${group.id}`}>
```

File: awx/ui_next/src/screens/Inventory/InventoryHosts/InventoryHostList.test.jsx [hunk 4]
```diff
@@ -196,7 +209,12 @@ describe('<InventoryHostList />', () => {
 
   test('delete button is disabled if user does not have delete capabilities on a selected host', async () => {
     await act(async () => {
-      wrapper.find('DataListCheck[id="select-host-3"]').invoke('onChange')();
+      // wrapper.find('DataListCheck[id="select-host-3"]').invoke('onChange')();
```
Comment (marshmalien) [inline awx/ui_next/src/screens/Inventory/InventoryHosts/InventoryHostList.test.jsx]:
```suggestion
```

**Comment by nixocio** [comment]
I triggered another e2e test run.

**Review by nixocio — APPROVED** [review]

**Review by tiagodread — APPROVED** [review]
All inventory sub lists were migrated to tables except for ` Inventories -> Inventory -> Groups tab -> Group -> Related group tab list and Hosts tab list` that was added as an item in the main checklist issue. 

Fixed broken e2e tests in 6300

**Pull request merged**